src/eventSchemas/models/schemaCodeLangs.ts

```diff
@@ -9,9 +9,10 @@ import { Set as ImmutableSet } from 'immutable'
 export const JAVA = 'Java 8+'
 export const PYTHON = 'Python 3.6+'
 export const TYPESCRIPT = 'Typescript 3+'
-export type SchemaCodeLangs = 'Java 8+' | 'Python 3.6+' | 'Typescript 3+'
+export const GOLANG = 'Go 1+'
+export type SchemaCodeLangs = 'Java 8+' | 'Python 3.6+' | 'Typescript 3+' | 'Go 1+'
```

Comment:
Nit: I'm assuming that the consts are used largely to define the `SchemaCodeLangs` type? If so, convert to `export type SchemaCodeLangs = typeof JAVA | typeof PYTHON | typeof TYPESCRIPT | typeof GOLANG`